.github/workflows/vulnz-docker-release.yml

```
@@ -0,0 +1,46 @@
+name: docker pr
+
+on:
+  push:
+    tags:
+      - vulnz/*
```

The entire repo is tagged on each new release with `v#.#.#`. As such, the tag config should be:
```suggestion
      - v*
```